tutorials/applications/Shadow estimation for VQS.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        N = 20
</pre><p>What is N?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I wrote down the theoretical number of shadow size per cluster on the line above, but this makes simulations run too slowly on a laptop. After some trial and error I figured <code>N=20</code>  was good enough.</p>